freqtrade/templates/base_strategy.py.j2

```diff
@@ -129,6 +129,13 @@ class {{ strategy }}(IStrategy):
                 (dataframe['volume'] > 0)  # Make sure Volume is not 0
             ),
             'enter_long'] = 1
+        # Uncomment to use shorts (remember to set margin in trading mode in configuration)
```

Comment:
i'd rather point this to the documentation
margin is not supported (futures is) - but it may require additional configuration - not just the flick of a flag.